.changeset/purple-parrots-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+
+---
```

Comment:
@Rhotimee These empty changelogs are not used. If the root package changed, there is no need to create a changeset.

Reply:
Oh. okay, noted 👍🏽